### PR TITLE
openapi(engine): improve OpenAPI readablity

### PR DESCRIPTION
**Pull request by lance6716**
Signed-off-by: lance6716 <lance6716@gmail.com>

### What problem does this PR solve?

Issue Number: ref #6817

### What is changed and how it works?

the visual change can be verified in `engine/jobmaster/dm/api_test.go`

#### Tests 

 - Unit tests

### Release note 

```release-note
 `None`.
```

### CODE REVIEW

File: engine/executor/dm/worker.go [hunk 1]
```diff
@@ -215,7 +215,7 @@ func (w *dmWorker) tryUpdateStatus(ctx context.Context) error {
 	if currentStage == previousStage {
 		return nil
 	}
-	w.Logger().Info("task stage changed", zap.String("task-id", w.taskID), zap.Int("from", int(previousStage)), zap.Int("to", int(currentStage)))
+	w.Logger().Info("task stage changed", zap.String("task-id", w.taskID), zap.String("from", string(previousStage)), zap.String("to", string(currentStage)))
```
Comment (GMHDBJD) [inline engine/executor/dm/worker.go:218]: Can we use zap.Stringer?

**Review by GMHDBJD — APPROVED** [review]
LGTM

File: engine/executor/dm/worker.go (hunk 1 above)
Comment (lance6716) [reply engine/executor/dm/worker.go:218]: I haven't added String() for this type

File: engine/jobmaster/dm/api.go [hunk 2]
```diff
@@ -29,17 +29,17 @@ import (
...
+	ExpectedStage  metadata.TaskStage         `json:"expected_stage"`
+	WorkerID       frameModel.WorkerID        `json:"worker_id"`
+	ConfigOutdated bool                       `json:"config_outdated"`
+	Status         *dmpkg.QueryStatusResponse `json:"status"`
 }
 
 // JobStatus represents status of a job
 type JobStatus struct {
-	JobID frameModel.MasterID
+	JobID frameModel.MasterID `json:"job_id"`
```
Comment (amyangfei) [inline engine/jobmaster/dm/api.go:40]: Is there any regulation that we follow about using `hyphen` or `underscore` in json key.

**Review by amyangfei — APPROVED** [review]

**Pull request merged**